### PR TITLE
Ginkgo- Fix issues with logs

### PR DESCRIPTION
**Pull request by eloycoto**
Fix issues that we had on the jenkins builds from 334 to 341. Issue
introduced by commit ca0dbdf96e6f7331df091e7ffbfe622b521f220e.

Signed-off-by: Eloy Coto <eloy.coto@gmail.com>

### CODE REVIEW

File: test/k8sT/Tunnels.go [hunk 1]
```diff
@@ -30,12 +30,13 @@ var _ = Describe("K8sTunnelTest", func() {
...
 	var demoDSPath string
 	var initialized bool
+	var logger *logrus.Entry
 
 	initialize := func() {
 		if initialized == true {
 			return
 		}
-		logger := log.WithFields(logrus.Fields{"testName": "K8sTunnelTest"})
+		logger := logrus.WithFields(logrus.Fields{"testName": "K8sTunnelTest"})
```
Comment (raybejjani) [inline test/k8sT/Tunnels.go]: There is a package scoped `log` object (that is a `*logrus.Logger` type) that you can use. If you use the package level function you get the default logrus logger as the base, and this is the one that receives all the library debug messages we were trying to exclude. Also, I think the `:=` shadows the more global variable which you added here.
tl;dr `logger = log.WithFields(logrus.Fields{"testName": "K8sTunnelTest"})` (note I dropped the `:=`

File: test/runtime/Policies.go [hunk 2]
```diff
@@ -56,7 +57,7 @@ var _ = Describe("RuntimePolicyEnforcement", func() {
 		if initialized == true {
 			return
 		}
-		logger = log.WithFields(log.Fields{"testName": "RuntimePolicyEnforcement"})
+		logger = logrus.WithFields(logrus.Fields{"testName": "RuntimePolicyEnforcement"})
```
Comment (raybejjani) [inline test/runtime/Policies.go]: Oops! this was definitely a mistake. You still want to use the package local `log` variable (in log.go) but `logrus.Fields` is the correct thing, and you'd need to import the package as you have.

File: test/runtime/Policies.go [hunk 3]
```diff
@@ -374,7 +375,7 @@ var _ = Describe("RuntimePolicies", func() {
 		if initialized == true {
 			return
 		}
-		logger = log.WithFields(log.Fields{"test": "RunPolicies"})
+		logger = logrus.WithFields(logrus.Fields{"test": "RunPolicies"})
```
Comment (raybejjani) [inline test/runtime/Policies.go]: same as the comment above, `log.WithFields` but `logrus.Fields`

File: test/test_suite_test.go [hunk 4]
```diff
@@ -48,9 +48,9 @@ func init() {
 		os.Exit(-1)
 	}
 
-	log.SetOutput(GinkgoWriter)
-	log.SetLevel(log.DebugLevel)
-	log.SetFormatter(&log.TextFormatter{
+	logrus.SetOutput(GinkgoWriter)
+	logrus.SetLevel(logrus.DebugLevel)
+	logrus.SetFormatter(&logrus.TextFormatter{
```
Comment (raybejjani) [inline test/test_suite_test.go]: Of these 3, SetOutput and SetFormatter are only possible on the logrus package (https://github.com/cilium/cilium/commit/ca0dbdf96e6f7331df091e7ffbfe622b521f220e#diff-599b7f4143f4fea0dcb52272ada9be3aR110) but SetLevel can be called on the log object. Calling it on the logrus package logger changes the log level for the same logger used by libraries, and they are noisy in debug. I'd suggest making a package variable called log (`log = common.DefaultLogger` is how I was doing it elsewhere) and then call SetLevel on it directly. The other places we try to log would then keep using `log.`.
You might want to suppress more of the logrus default logger messages by changing that level (see https://github.com/cilium/cilium/commit/ca0dbdf96e6f7331df091e7ffbfe622b521f220e#diff-599b7f4143f4fea0dcb52272ada9be3aR121)

**Review by raybejjani — CHANGES_REQUESTED** [review]
I'm really sorry I didn't test this! I thought I tried the various things we did :(

Some things you don't need to change, unless you want to use the logrus package logger. I recommend against it mostly to stay consistent. When I made the changes in the PR that broke this, I wasn't sure where in the CI the most noisy messages were. We could use the logrus default in some cases, if there is a strong desire to.
Assuming you don't want the logrus default, the general thing missing here was that:
- I removed logrus imports in places where we needed it for logrus.Fields
- I didn't create a log logrus.Logger object in scope for some packages.
- I used SetOutput and SetFormatter on the Logger object but they are package only functions in our version of logrus.

These are why you saw the build errors.

**Review by raybejjani — APPROVED** [review]
